src/main/java/uk/gov/hmcts/reform/sendletter/data/LetterRepository.java

```diff
@@ -65,9 +64,12 @@ public void save(DbLetter letter, Instant creationTime, String messageId) throws
      * @param serviceName String
      * @return Letter status.
      */
-    public Optional<LetterStatus> getLetterStatus(UUID id, String serviceName) {
+    public Optional<uk.gov.hmcts.reform.sendletter.model.out.LetterStatus> getLetterStatus(
```

Comment:
was this change intentional? now repo layer is returning api layer model

Reply:
Nothing changed, but there are two `LetterStatus` classes used here now and one of them has be referred to its fully qualified name